### PR TITLE
Error log cleanup

### PR DESCRIPTION
**Pull request by raxo2222**
Some more log error fixing

### CODE REVIEW

**Review by JackEPepper — APPROVED** [review]

File: common/building_groups/00_building_groups.txt
Comment (DZappala) [inline common/building_groups/00_building_groups.txt:1]: Should slums have cash reserves?
Set to 0 (or 1 if 0 is not possible)

Or do buildings _need_ cash reserves in order for them to work? @JackEPepper @raxo2222

File: common/character_traits/cwp_traits.txt
Comment (DZappala) [inline common/character_traits/cwp_traits.txt:1]: Add the following line. 

```
#TODO create events and/or conditionals to allow lawyer characters to have some sway on passing laws [for 1.4 DLC release]
```

**Review by DZappala — CHANGES_REQUESTED** [review]
Take a look at comment 😃

File: common/character_traits/cwp_traits.txt
Comment (raxo2222) [reply common/character_traits/cwp_traits.txt:1]: Added

File: localization/english/triggers_l_english.yml [hunk 1]
```diff
@@ -534,8 +534,8 @@
  TRIGGER_STANDARD_OF_LIVING:0 "Standard of living $COMPARATOR$ #variable $NUM$#!"
  TRIGGER_STANDARD_OF_LIVING_NOT:1 "Standard of living #bold not #!$COMPARATOR$ #variable $NUM$#!"
  
- TRIGGER_FULFILLMENT:0 "Fulfillment $COMPARATOR$ #variable $NUM$#!"
- TRIGGER_FULFILLMENT_NOT:1 "Fulfillment #bold not #!$COMPARATOR$ #variable $NUM$#!"
+ TRIGGER_FULLFILLMENT:0 "Fulfillment $COMPARATOR$ #variable $NUM$#!"
```
[the remark this answers is not in the document]
Comment (raxo2222) [reply localization/english/triggers_l_english.yml:537]: Interestingly its not used in \game\common\trigger_localization\00_trigger_localization.txt unlike others
I guess we can use correct spelling until we get a loc error

[the remark this answers is not in the document]
Comment (DZappala) [reply localization/english/triggers_l_english.yml]: Haha oops Disappearance "-ance" is the correct spelling. Idk what I was smoking at the time.

**Pull request merged**